doc/arkode/guide/source/Mathematics.rst

```diff
@@ -604,129 +604,231 @@ an identity mass matrix, :math:`M(t)=I`. The slow time scale may consist of only
 :numref:`ARKODE.Usage.MRIStep.CustomInnerStepper`).
 
-The final abscissa is :math:`c^S_{s+1}=1` and the coefficients
-:math:`\omega_{i,j}` and :math:`\gamma_{i,j}` are polynomials in time that
-dictate the couplings from the slow to the fast time scale; these can be
-expressed as in :cite:p:`ChiRen:21` and :cite:p:`Sandu:19` as
+Below we summarize the details for each method family. For additional
+information, please see the references listed above.
 
-.. math::
-   \omega_{i,j}(\tau) = \sum_{k\geq 0} \omega_{i,j}^{\{k\}} \tau^k
-   \quad\text{and}\quad
-   \gamma_{i,j}(\tau) = \sum_{k\geq 0} \gamma_{i,j}^{\{k\}} \tau^k,
-   :label: ARKODE_MRI_coupling
 
-and where the tables :math:`\Omega^{\{k\}}\in\mathbb{R}^{(s+1)\times(s+1)}` and
-:math:`\Gamma^{\{k\}}\in\mathbb{R}^{(s+1)\times(s+1)}` define the slow-to-fast
-coupling for the explicit and implicit components respectively.
+MIS, MRI-GARK, and IMEX-MRI-GARK Methods
+----------------------------------------
 
-For traditional MIS methods, the coupling coefficients are uniquely defined
-based on a slow Butcher table :math:`(A^S,b^S,c^S)` having an explicit first
-stage (i.e., :math:`c^S_1=0` and :math:`A^S_{1,j}=0` for :math:`1\le j\le s`),
-sorted abscissae (i.e., :math:`c^S_{i} \ge c^S_{i-1}` for :math:`2\le i\le s`),
-and the final abscissa is :math:`c^S_s \leq 1`. With these properties met, the
-coupling coefficients for an explicit-slow method are given as
+The methods in IMEX-MRI-GARK family, which includes MIS and MRI-GARK methods,
+are defined by a vector of slow stage time abscissae, :math:`c^S \in
+\mathbb{R}^{s}`, and a set of coupling tensors,
+:math:`\Omega\in\mathbb{R}^{(s+1)\times s \times k}` and
+:math:`\Gamma\in\mathbb{R}^{(s+1)\times s \times k}`, that specify the
+slow-to-fast coupling for the explicit and implicit components, respectively.
+
+The fast stage IVPs, :eq:`MRI_fast_IVP`, are evolved over non-overlapping
+intervals :math:`[t_{0,i},t_{F,i}] = [t_{n,i-1}^S, t_{n,i}^S]` with
+the initial condition :math:`v_{0,i}=z_{i-1}`. The fast IVP forcing function is
+given by
 
 .. math::
-   \omega_{i,j}^{\{0\}} = \begin{cases}
-   0, & \text{if}\; i=1,\\
-   A^S_{i,j} - A^S_{i-1,j}, & \text{if}\; 2\le i\le s,\\
-   b^S_j - A^S_{s,j}, & \text{if}\; i=s+1.
-   \end{cases}
-   :label: ARKODE_MIS_to_MRI
+   r_i(t) = \frac{1}{\Delta c_i^S} \sum\limits_{j=1}^{i-1} \omega_{i,j}(\tau) f^E(t_{n,j}^S, z_j)
+   + \frac{1}{\Delta c_i^S} \sum\limits_{j=1}^i \gamma_{i,j}(\tau) f^I(t_{n,j}^S, z_j)
 
-For general slow tables :math:`(A^S,b^S,c^S)` with at least second-order
-accuracy, the corresponding MIS method will be second order. However, if this
-slow table is at least third order and satisfies the additional condition
+where :math:`\Delta c_i^S=\left(c^S_i - c^S_{i-1}\right)`, :math:`\tau = (t -
+t_{n,i-1}^S)/(h^S \Delta c_i^S)` is the normalized time, the coefficients
+:math:`\omega_{i,j}` and :math:`\gamma_{i,j}` are polynomials in time of degree
+:math:`k` given by
```

Comment:
In this section k is used as both the degree (a fixed constant) and an index variable which could be a bit confusing. Using two variables may help